core/spec/models/spree/promotion_spec.rb

```diff
@@ -384,6 +384,73 @@
     end
   end
 
+  context '#unstarted?' do
```

Comment:
_(Still splitting hairs)_ I think this should be more appropriately a `describe` instead of a `context`. I understand this file uses both methods indifferently, still `describe` sounds more appropriate to me when referring to a method.

Reply:
I will update it, I was trying to keep the file convention. Thanks for the feedback 😄